src/main/java/li/cil/oc2/common/Config.java

```diff
@@ -39,6 +39,7 @@
     @Path("energy.items") public static int networkTunnelEnergyPerTick = 2;
 
     @Path("gameplay") public static ResourceLocation blockOperationsModuleToolTier = TierSortingRegistry.getName(Tiers.DIAMOND);
+    @Path("gameplay.devices") public static long soundCardCoolDownSeconds = 2;
```

Comment:
I'd just throw this directly into `gameplay` for now, since the "soundCard" is part of the config name. Maybe at some point in the future we'll have to make this more granular, but for now I'd keep it simple.